src/main/kotlin/br/com/fiap/postech/foodchallenge/domain/entities/order/Order.kt

```diff
@@ -0,0 +1,31 @@
+package br.com.fiap.postech.foodchallenge.domain.entities.order
+
+import br.com.fiap.postech.foodchallenge.domain.entities.CPF
+import br.com.fiap.postech.foodchallenge.infrastructure.persistence.entities.OrderEntity
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.treeToValue
+import java.time.LocalDateTime
+
+data class Order(
+    val id: Long? = null,
+    val customerCpf: CPF? = null,
+    val items: List<OrderItem>,
+    val status: OrderStatus,
+    val createdAt: LocalDateTime
```

Comment:
aqui eu acho melhor ficar como default da propriedade `createdAt` o valor de agora, deixando a declaração assim: `val createdAt: LocalDateTime = LocalDateTime.now()`

assim toda vez que um objeto de domínio for instanciado, mesmo que seja pelo nosso método estático de fábrica, não precisamos nos preocupar se estamos passando a data certa ou não